### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mary-grace**
Updated to distinguish between DevEx and DevRel.

### CODE REVIEW

**Comment by mary-grace** [comment]
@akeller Could you evaluate this from a DevEx standpoint:
* does the statement about your team make sense?
* is this still your team's mission (pulled from the Handbook)
* are the roles represented on your team correct?

@xomiamoore Could you evaluate this from a Community Hub perspective:
* is this information useful to our community members?
* does it accurately represent our team in a way that is valuable to those who would be reading it?

Thanks!

**Review by akeller — APPROVED** [review]
Looks great! And thanks for finding my very recent updates for the team mission 😄

File: README.md [hunk 1]
```diff
@@ -1,35 +1,28 @@
...
 
-The Camunda Developer Relations team exists to serve the Camunda developer community, from internal employees to Open Source contributors, Enterprise customers, and the broader tech community.
+We're a globally distributed team made up of developers and tech-savvy individuals who care about the tech community. We enjoy creating diverse and inclusive communities that provide solutions for our technical audiences. We have two distinct teams:
+* **Developer Relations** focuses on creating a valuable community experience for Camunda users and encouraging external advocates to spread the word about Camunda. We do this by ensuring people can find solutions to their problems through the forum & content, listening to the feedback from our community and passing it along to the proper teams, and making it easy for community members to contribute code as well as content.
+* **Developer Experience** focuses on external developer enablement for products through documentation and product-adjacent tools at Camunda. Our mission is to lower barriers for all developers working programmatically in and around the Camunda ecosystem to get what they need to be effective with Camunda efficiently.
```
Comment (akeller) [inline README.md:7]:
```suggestion
* **[Developer Relations](https://github.com/orgs/camunda-community-hub/teams/devrel)** focuses on creating a valuable community experience for Camunda users and encouraging external advocates to spread the word about Camunda. We do this by ensuring people can find solutions to their problems through the forum & content, listening to the feedback from our community and passing it along to the proper teams, and making it easy for community members to contribute code as well as content.
* **[Developer Experience](https://github.com/orgs/camunda-community-hub/teams/devex)** focuses on external developer enablement for products through documentation and product-adjacent tools at Camunda. Our mission is to lower barriers for all developers working programmatically in and around the Camunda ecosystem to get what they need to be effective with Camunda efficiently.
```

Comment (akeller) [reply README.md:7]: @mary-grace I don't know if you want to link to the teams here or at the bottom.

**Comment by xomiamoore** [comment]
Thank you both!

**Pull request merged**